src/fprime_gds/executables/run_deployment.py

```diff
@@ -13,9 +13,18 @@
     GdsParser,
     ParserBase,
     StandardPipelineParser,
+    OpenMCTTelemetryPollerParser
 )
 from fprime_gds.executables.utils import AppWrapperException, run_wrapped_application
 
+# Try to Import FPrime-OpenMCT Python Packages
+try: 
+    from fprime_openmct.config_server import ServerConfig
+    from fprime_openmct.fprime_to_openmct import TopologyAppDictionaryJSONifier
+except:
+    pass
```

Comment:
This except should be restricted to only catch import errors, and leave a flag behind e.g. `OPENMCT_NOT_INSTALLED=True` that you can check below...

Reply:
should I just have `import openmct` in the try statement then? Something like:
```
try:

     import openmct

except:

     OPENMCT_INSTALLED = False 
```
And then have the ServerConfig and TopologyAppDictionaryJSONifier imports in a conditional statement after?

Reply:
Yeah you can actually do

```
try:
    import openmct
except ImportError:
    openmct = None
```

Then you can simply use 

```
if openmct:
     use openmct.ServerConfig() etc...
else:
    raise <error ....>
```

Reply:
Sounds good! I've followed this structure in my most recent commits